samples/cert-app/main.c

```diff
@@ -193,13 +221,17 @@ int main(int argc, char **argv)
 	printf("    - Welcome to librats sample cert-app program for Host\n");
 #endif
 
-	char *const short_options = "dhkc:";
+	char *const short_options = "dhkc:a:v:p:l:";
 	// clang-format off
     struct option long_options[] = {
         { "debug-enclave", no_argument, NULL, 'd' },
         { "help", no_argument, NULL, 'h' },
         { "no-privkey", no_argument, NULL, 'k'},
         { "add-claim", required_argument, NULL, 'c' },
+	{ "attester", required_argument, NULL, 'a' },
+        { "verifier", required_argument, NULL, 'v' },
+	{ "crypto", required_argument, NULL, 'p' },
```

Comment:
Would be better to align with rats-tls here. What about using `-c` for `crypto` and `-C` for `add-claim`?

Reply:
Modified

Reply:
OK, -C stands for add-claim, -c stands for crypto